### PR TITLE
Added more documentation on solvepnp

### PR DESCRIPTION
**Pull request by neel-j-dev**
Added more information on what the camera matrix and distortion coefficients mean.
Added the point of calibrating and the different types of distortion
Added what the NT values for targetPose mean
Camera pitch and good practices for calibrating

### CODE REVIEW

File: docs/source/advanced/solvePNP.rst [hunk 1]
```diff
@@ -5,14 +5,97 @@ SolvePNP
...
+   - .. image:: /images/solvepnp/camera_matrix.PNG
+   - **fx** and **fy** : These are the focal lengths of the camera expressed in pixels
+   - **cx** and **cy** : These are the principal points and refer to the image center 
+
+- Distortion Coefficents
+
+   - distortion coefficents are vectors of up to 8 parameters, formated like : **k1, k2, p1, p2 [, k3 [, k4, k5, k6]]**
+   - **k1, k2..** are radical distortion coefficents
+   - **p1, p2...** are tangential distortion coefficents
+   - The distortion do not change based off the camera resolution, so the distortion coefficents will be the same for 640 x 480 images and 320 x 240 images
```
Comment (modelmat) [inline docs/source/advanced/solvePNP.rst:62]: Does the `Camera Matrix` change based on camera resolution?

Also please indicate that this is an example, not an all-inclusive list.

File: docs/source/advanced/solvePNP.rst [hunk 2]
```diff
@@ -5,14 +5,97 @@ SolvePNP
...
+   - -targetwidth/2, -targetheight/2, 0
+   - targetwidth/2, -targetheight/2, 0
+   - targetwidth/2, targetheight/2, 0
+
+These world coordinates for the 2020 season are premade for the Loading Bay and Upper port in the dropdown menu or can
+be uploaded as a .csv file.
+
+Now that we have callibrated the camera, and found the 3D world coordinates, we can now use this information to:
+
+- Undisort an image recieved from the camera
```
Comment (modelmat) [inline docs/source/advanced/solvePNP.rst]:
```suggestion
- Undistort an image received from the camera
```

File: docs/source/advanced/solvePNP.rst [hunk 3]
```diff
@@ -5,14 +5,97 @@ SolvePNP
...
+- Returns a re-projection .. error
+
+OpenCV calculates reprojection error by projecting three-dimensional of chessboard points into the image using the final set of calibration parameters and comparing the position of the corners. An RMS error of 300 means that, on average, each of these projected points is 300 px away from its actual position.
+The closer the error is to 0, the better the callibration. 
+
+Before enabling 3D tracking, calculate the pitch between the horizontal axis and the optical axis of your camera and enter
+it into the **Camera Pitch** field in the chameleon UI.
+
+.. image:: /images/solvepnp/pitch.PNG
+   :alt: Source: Research Gate
```
Comment (modelmat) [inline docs/source/advanced/solvePNP.rst]: not exactly the point of alt text, and it's very vague as well.

Comment (neel-j-dev) [reply docs/source/advanced/solvePNP.rst]: Alright took it out then

**Comment by XzibitGG** [comment]
Yeah, camera matrices change proportionally to changes in resolution, added that in.

File: docs/source/advanced/solvePNP.rst (hunk 1 above)
Comment (neel-j-dev) [reply docs/source/advanced/solvePNP.rst:62]: Yeah, camera matrices change proportionally to changes in resolution, added that in.

File: docs/source/advanced/solvePNP.rst [hunk 4]
```diff
@@ -5,14 +5,99 @@ SolvePNP
...
+After you complete the callibration process, on the command line, you'll see:
+
+- Camera Matrix
+
+   - The camera matrix is a 3 by 3 matrix as show below
+   - .. image:: /images/solvepnp/camera_matrix.PNG
+   - **fx** and **fy** : These are the focal lengths of the camera expressed in pixels
+   - **cx** and **cy** : These are the principal points and refer to the image center 
+   - The camera matrix will be scaled proportionally to any change in resolution. For example, if resolution goes from 640 x 480 to 320 x 240 images
+
```
Comment (oriagranat9) [inline docs/source/advanced/solvePNP.rst]: you have to calibrate for each resolution we dont scale

File: docs/source/advanced/solvePNP.rst [hunk 5]
```diff
@@ -5,14 +5,99 @@ SolvePNP
...
+   
+      - **fx** and **cx**: These will be multiplied by (new resolution x / old resolution x) or in the example, (320 / 640)
+      - **fy** and **cy**: These will be multiplied by (new resolution y / old resolution y) or in the example, (240 / 480)
+
+- Distortion Coefficents
+
+   - distortion coefficents are vectors of up to 8 parameters, formated like : **k1, k2, p1, p2 [, k3 [, k4, k5, k6]]**
+   - **k1, k2..** are radical distortion coefficents
+   - **p1, p2...** are tangential distortion coefficents
+   - The distortion do not change based off the camera resolution, so the distortion coefficents will be the same for 640 x 480 images and 320 x 240 images
```
Comment (oriagranat9) [inline docs/source/advanced/solvePNP.rst:62]: same here for calibration

File: docs/source/advanced/solvePNP.rst (hunk 4 above)
Comment (neel-j-dev) [reply docs/source/advanced/solvePNP.rst]: Want me to keep the explanation and write underneath that you'll need to recalibrate or scrap it and write it in place?

Comment (oriagranat9) [reply docs/source/advanced/solvePNP.rst]: except that everything is perfect i think you can just remove that sentence

Comment (neel-j-dev) [reply docs/source/advanced/solvePNP.rst]: Alright, done

**Review by oriagranat9 — APPROVED** [review]

**Comment by XzibitGG** [comment]
Doesn't look like the images got merged as well.

**Pull request merged**